openmetadata-ui/src/main/resources/ui/src/pages/ServiceDetailsPage/ServiceDetailsPage.tsx

```diff
@@ -672,7 +672,7 @@ const ServiceDetailsPage: FunctionComponent = () => {
 
         setServiceDetails(response);
       } catch (err) {
-        // Error
+        showErrorToast(err as AxiosError);
```

Comment:
Don't think we want to show this error, We will be showing noDataPlaceHolder in case we failed to fetch details no?

Reply:
@chirag-madlani This is a `Patch API`, so we need to show the message to the user if the request fails.